tests/ut/backends/test_memory.py

```diff
@@ -198,3 +198,9 @@ def test_default_serializer(self):
 
     def test_parse_uri_path(self):
         assert SimpleMemoryCache().parse_uri_path("/1/2/3") == {}
+
+    def test_custom_serializer(self, valkey_config):
```

Comment:
This is test_memory.py. Probably the cause of the import failure.

```suggestion
    def test_custom_serializer(self):
```

Reply:
Good catch!